python/tk_multi_loader/delegate_publish_history.py

```diff
@@ -17,6 +17,107 @@
 shotgun_model = sgtk.platform.import_framework("tk-framework-shotgunutils", "shotgun_model")
 shotgun_view = sgtk.platform.import_framework("tk-framework-qtwidgets", "shotgun_view")
 
+from .ui.widget_publish_history import Ui_PublishHistoryWidget
+
+class PublishHistoryWidget(QtGui.QWidget):
```

Comment:
This is how it started out - that was my gut feeling too - but it actually felt better once i had moved it inside this file - because the delegate and the widget are so related. I can move it out if you feel strongly about it, but in my opinion this would be a case where a good general rule of thumb is deliberately broken for improved readability! :)